### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by Tyler-V**

### CODE REVIEW

File: VirtualLibrary/Program.cs [hunk 1]
```diff
@@ -1,125 +1,46 @@
...
-// Middleware
-// Each component performs operations on an HttpContext and either invokes the next middleware in the pipeline or terminates the request.
 app.UseHttpsRedirection();
 app.UseStaticFiles();
 
-// ROUTING
-// URL pattern that is mapped to a handler.
-
 app.UseRouting();
+app.UseAuthentication();;
```
Comment (Tyler-V) [inline VirtualLibrary/Program.cs]: ;;

File: VirtualLibrary/Data/ApplicationDbContext.cs [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
+        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options)
         {
+
+        }
+        public DbSet<Book> Book { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            base.OnModelCreating(modelBuilder);
+            modelBuilder.Entity<Book>().HasData(
```
Comment (Tyler-V) [inline VirtualLibrary/Data/ApplicationDbContext.cs:18]: Seeding data should be done in Program.cs outside of OnModelCreating

File: VirtualLibrary/Migrations/20220919144754_addIdentityToDb.cs [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
 using Microsoft.EntityFrameworkCore.Migrations;
-using System;
 
-namespace VirtualLibrary.Data.Migrations
+#nullable disable
+
+namespace VirtualLibrary.Migrations
 {
-    public partial class CreateIdentitySchema : Migration
+    public partial class addIdentityToDb : Migration
```
Comment (Tyler-V) [inline VirtualLibrary/Migrations/20220919144754_addIdentityToDb.cs:8]: Name migrations consistently

File: VirtualLibrary/Program.cs (hunk 1 above)
Comment (pm24mcgi) [reply VirtualLibrary/Program.cs]: fixed

File: VirtualLibrary/Data/ApplicationDbContext.cs (hunk 2 above)
Comment (pm24mcgi) [reply VirtualLibrary/Data/ApplicationDbContext.cs:18]: Finding it difficult to find the correct guidance on how to complete this. Everything I read is utilizing context/page model to implement seed data.

File: VirtualLibrary/Migrations/20220919144754_addIdentityToDb.cs (hunk 3 above)
Comment (pm24mcgi) [reply VirtualLibrary/Migrations/20220919144754_addIdentityToDb.cs:8]: Ok

**Pull request merged**